crates/linker/src/lib.rs

```diff
@@ -923,11 +936,11 @@ pub fn preprocess(
 fn gen_macho_le(
     exec_data: &[u8],
     md: &mut metadata::Metadata,
-    out_filename: &str,
+    out_filename: &Path,
     macho_load_so_offset: usize,
     _target: &Triple,
     _verbose: bool,
-) -> (MmapMut, File) {
```

Comment:
we don't need to hold onto the file

Reply:
Interesting. I guess I just missed that in some refactor that pulled this function out.

Reply:
I think I did this originally, thinking that not dropping the file was relevant. But it isn't, so we can get rid of it